README.md

```diff
@@ -1,26 +1,23 @@
-Puppet Labs Spec Helper
-=======================
+# Puppet Labs Spec Helper
```

Comment:
I generally feel (as a non-owner of the repo) that non-change diffs should be removed from a PR so as to focus on the changes. A separate PR with the README improvements would make review easier.

Reply:
Removed automated conversion as suggested.